### ISSUE TITLE
Safari 5.1.7 does not work

### ISSUE THREAD

**Issue by tresf** [qzind/qz-print#17]
When attempting to use QZ Tray on OSX Lion, Safari 5.1.7 never establishes a socket connection and Jetty shows the following error:

```
2015-04-17 18:21:13.157:WARN:oejws.WebSocketServerFactory:qtp1956710488-25: Client 0:0:0:0:0:0:0:1 (:49256) User Agent: [unset] requested WebSocket version [-1], Jetty supports version: [13]
2015-04-17 18:21:13.348:WARN:oejh.HttpParser:qtp1956710488-25: badMessage: 400 for HttpChannelOverHttp@30a8f8d8{r=1,c=false,a=IDLE,uri=-}
```

**Comment by tresf** [comment]
From what I'm reading, Safari 5.x supports a very old version of WebSockets.  We may want to capture this version requirement message for end-users so that they are notified when on older browsers.

**Comment by akberenz** [comment]
@tresf I have a possible fix, but I need you to confirm that it won't give a false negative on newer versions of safari, which I cannot download for windows.
Can you post the output of `console.log(typeof WebSocket);` from a newer version of safari?

**Comment by tresf** [comment]
``` html
<html>
<script>console.log(typeof WebSocket);</script>
</html>
```

Safari 8.0.4, OSX Yosemite (2015):

```
object
```

Safari 5.1.7, OSX Lion (2012):

```
object
```

**Comment by akberenz** [comment]
Just want to document the source for this fix:
http://stackoverflow.com/questions/17849517/check-to-see-if-websocket-is-hixie-client-side

**Comment by tresf** [comment]
Terrific.  We'll consider this closed via 8912b4f. :)
